### DISCUSSION TITLE
rules_python version compatibility with Bazel 4.2.2

### DISCUSSION THREAD

**Discussion by harshtrivedi134** [bazel-contrib/rules_python#1941] [Q&A]
I'm running an old verison of bazel 4.2.2
I understand that [support matrix](https://bazel.build/release#support-matrix), does not recommend Bazel 4.x.x. However,  upgrading bazel version would require upgrading my protobuf version and that would pose many challenges. 
What version of `rules_python` do you recommend for my case?
 

Thank you!

**Comment by aignas** [comment]
You can look at the release dates and select the rules_python version that was released before bazel 6 was released. We tend to maintain support of two bazel versions at all times.
